### ISSUE TITLE
Implement 3-lobed lanczos filter for Image > Image Size menu

### ISSUE THREAD

**Issue by sethborg** [photopea/photopea#6796]
I noticed that when downscaling an image with photopea the result looks worse (blurry) than if I just use the original size image and let the browser scale it. I looked into what firefox uses. Instead of bilinear filtering and it's called "3-lobed lanczos filter"  https://entropymine.com/resamplescope/notes/browsers/

Could photopea implement this on the Image > Image Size menu?

Steps to reproduce
1. Take a logo that has some text on it like https://upload.wikimedia.org/wikipedia/commons/thumb/8/81/U.S._Department_of_State_official_seal.svg/1200px-U.S._Department_of_State_official_seal.svg.png this is a 1200x1200 pixel image. Put it in an image tag with width and height of like 150x150

2. Use photopea to resize the 1200x1200 image to a 150x150 image

3. Create an image tag for the photo resized with photopea and leave it at it's native size

4. Compare the two images side by side to see the photopea resized image is significantly more blurry

<img width="212" alt="example" src="https://github.com/photopea/photopea/assets/6662944/c5de3b55-b2fa-45ba-924a-1a6df2d341ea">

**Comment by photopea** [comment]
I have made this demo: https://jsfiddle.net/6hdw2m71/

I took a magnifying glass and counded the pixels on my monitor. The size of a logo is 225 monitor pixels. https://i.imgur.com/sgIP1Wy.png

Did you try to take a magnifying glass and count the pixels on your monitor? Because "150px" in HTML does not mean 150 pixels on your screen.

Here is my explanation of your two cases:
1) Your browser stretches a 150x150 pixel image from Photopea to 225x225 pixels (for the screen)
2) Your browser shrinks a 1200x1200 pixel image to 225x225 pixels (for the screen)

So you are comparing a 150x150 image to 225x225 image, and the second one is better, which is obvious.
